src/main.rs

```diff
@@ -3,6 +3,10 @@ extern crate colored;
 #[macro_use]
 extern crate error_chain;
 extern crate ethabi;
+#[macro_use]
+extern crate ethabi_derive;
+#[macro_use]
+extern crate ethabi_contract;
```

Comment:
I found out recently that you can import specific macros by name using:

```
#[macro_use(use_contract)]
extern crate ethabi_contract;
```
which I think adds a nice level of readability. In this specific case I don't think that making the change would add a huge amount of readability because `use_contract!` is so widely used in the Rust-Ethereum world and because it is the only macro in the `ethabi_contract` crate, but I think that it is a small code-style change 
that is ultimately beneficial. What do you think?

Reply:
Good idea, I agree!